### DISCUSSION TITLE
Use Traditional Pagination on Archive Pages Instead of the "Load More" Button

### DISCUSSION THREAD

**Discussion by tchocky21** [WebDevStudios/nextjs-wordpress-starter#1015] [Q&A]
Hi all - 

I'm looking for some guidance on creating a traditional numbered pagination component instead of using the "Load More" button that comes with the starter. The URL structure would be something like `category/[category_slug]/page/[number]`. I'm having trouble understanding how to use WPGraphQL's cursor based paging in conjunction with `getStaticProps` and `getStaticPaths`. 

Any suggestions on a starting point would be much appreciated. Thanks :)

**Comment by feroz39** [comment]
Any suggestions?
